packages/experimental/labs-workers/package.json

```diff
@@ -30,7 +29,7 @@
     "modelfusion": "^0.137.0"
   },
   "devDependencies": {
-    "@cloudflare/workers-types": "^4.20240423.0",
+    "@cloudflare/workers-types": "^4.20240502.0",
```

Comment:
we'd need to bump packages/apps/composer/app also to be in sync.

Reply:
good catch, updated this now!